integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -8538,6 +8550,84 @@ msgstr "Summe pro Art des Inhalts"
+msgstr ""
+"Offline-Zugriffe können nicht mitgerechnet werden. D.h. auch eine Seite mit "
+"wenigen Zugriffen in der Liste kann höhere Zugriffe haben, die offline "
+"stattgefunden haben."
+
+#: cms/templates/tutorials/page_access_statistics.html
+msgid ""
+"Pages that contain specific information about seasonal or one-time events (e."
+"g. events or election times) may receive higher traffic during certain "
+"periods. Outside these phases, traffic may be lower."
+msgstr ""
+"Seiten, die spezifische Informationen zu saisonalen oder einmaligen "
+"Ereignissen enthalten (z.B. Veranstaltungen oder Wahlzeiten), können in "
+"bestimmten Phasen höhere Zugriffszahlen aufweisen. Außerhalb dieser Phasen "
+"sind die Zugriffe eventuell niedriger."
+
+#: cms/templates/tutorials/page_access_statistics.html
+msgid ""
+"Page views should not be viewed in isolation, but always in the context of "
+"the overall communication strategy. A page with few hits can still be an "
+"important part of user guidance and information."
+msgstr ""
+"Seitenzugriffe sollten nicht isoliert betrachtet werden, sondern immer im "
+"Kontext der gesamten Kommunikationsstrategie. Eine Seite mit wenigen "
+"Zugriffen kann trotzdem ein wichtiger Bestandteil der Nutzerführung und "
+"Information sein."
+
+#: cms/templates/tutorials/page_access_statistics.html
+msgid ""
+"Some pages may be rarely visited, but are extremely important for specific "
+"target groups (e.g. newcomers, people with special needs). For example, "
+"information about special advisory services or integration offers could not "
+"be used by the general public, but could be indispensable for the target "
+"group."
+msgstr ""
+"Manche Seiten werden möglicherweise selten besucht, sind aber für "
+"spezifische Zielgruppen (z.B. Neuankömmlinge, Menschen mit besonderen "
+"Bedürfnissen) extrem wichtig. Zum Beispiel könnten Informationen über "
+"spezielle Beratungsdienste oder Integrationsangebote nicht von der breiten "
+"Masse genutzt werden, aber für die Zielgruppe unverzichtbar sein."
+
+#: cms/templates/tutorials/page_access_statistics.html
+msgid ""
+"Content that is no longer current or appears outdated can lead to a decrease "
+"in traffic. It is therefore important to regularly check whether content has "
+"been updated and how this affects the statistics. Low traffic numbers could "
+"indicate that the page is no longer perceived as relevant."
+msgstr ""
+"Inhalte, die nicht mehr aktuell sind oder veraltet erscheinen, können zu "
+"einem Rückgang der Zugriffe führen. Es ist daher wichtig, regelmäßig zu "
+"überprüfen, ob Inhalte aktualisiert wurden und wie dies die Statistiken "
+"beeinflusst. Niedrige Zugriffszahlen könnten darauf hinweisen, dass die "
+"Seite nicht mehr als relevant wahrgenommen wird."
+
+#: cms/templates/tutorials/page_access_statistics.html
+msgid "You can find more information about this in "
+msgstr "Mehr Informationen dazu finden Sie im"
```

Comment:
Seems like there is an extra space (or padding in an an anchor tag above) needed in German version 
![image](https://github.com/user-attachments/assets/c779145a-88df-4bc1-8dd7-1b8eef9005d5)